code/clue.js

```diff
@@ -1,24 +1,61 @@
 // CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// const mrGreen = {
+//   firstName: "Jacob",
+//   lastName: "Green",
+//   color: "green",
+//   description: "He has a lot of connections",
+//   age: 45,
+//   image: "assets/green.png",
+//   occupation: "Entrepreneur"
+// }
 
-const mrGreen = {
-  firstName: "Jacob",
-  lastName: "Green",
-  color: "green",
-  description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
-  occupation: "Entrepreneur"
+
+
+ function Suspect(firstName, lastName, color, description, age, occupation, image) {
+  this.firstName = firstName;
+  this.lastName  = lastName;
+  this.color = color;
+  this.description = description;
+  this.age = age;
+  this.occupation = occupation;
+  this.image = image;
 }
```

Comment:
You should be able to write this function like you did on `pickRoom`! That's the "new" way of writing functions, which is good to practice. But they do the same!